show/plugins/cisco-8000.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+#########################################################
+# Copyright 2021 Cisco Systems, Inc.
+# All rights reserved.
+#
+# CLI Extensions for show command
+#########################################################
+
+try:
+    import click
+    import subprocess
+    import yaml
+    from show import platform
+except ImportError as e:
+    raise ImportError("%s - required module not found" % str(e))
+
+PYTHON3 = '/usr/bin/python3'
+PLATFORM_PY = '/opt/cisco/bin/platform.py'
+
+def invoke(args):
+    if 'run' in dir(subprocess):
+        e = subprocess.run(args, shell=False)
+        if e.returncode:
+            click.echo('{} failed'.format(e.args))
+    else:
+        e = subprocess.call(args, shell=False)
+        if e:
+            click.echo('{} failed'.format(' '.join(args)))
+
+# Platform specific version invoked by generic show.version
+def version(verbose):
+    filespec = '/opt/cisco/etc/build_info.yaml'
+    with open(filespec) as f:
+        build_info = yaml.safe_load(f)
+    for d in build_info.get('sdk_versions', []):
+        for k,v in d.items():
+            click.echo('{}: {}'.format(k,v))
+
+@click.group()
+def npu():
```

Comment:
Since it is a platform specific plugin, it should be under the platform type grouping ( Ref:  https://github.com/Azure/sonic-utilities/blob/master/show/plugins/mlnx.py )

Reply:
Fixed the code based on your recommendation